mindjunkies/forums/models.py

```
@@ -84,7 +85,7 @@ class LikedComment(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     
     def __str__(self):
-        return f'{self.user.username} : {self.comment.body[:30]}'
+        return f'{self.user.username} : {self.comment.body[:30]}' 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)